test-base.cfg

```diff
@@ -247,6 +249,53 @@ initialization =
     if len(sys.argv) < 2: sys.argv.extend([pkgdir])
     sys.argv.extend(${pydocstyle:args})
 
+# Downloads the default isort.cfg.
+[isort-cfg]
+recipe = collective.recipe.shelloutput
+cfgfile = ${buildout:directory}/parts/isort/.isort.cfg
+cfgurl = https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/.isort.cfg
+commands =
+    cmd1 = mkdir -p `dirname ${isort-cfg:cfgfile}`
+    cmd2 = curl "${isort-cfg:cfgurl}" > "${isort-cfg:cfgfile}"
+
+# bin/isort : isort validation of the packages source.
+[isort]
+recipe = zc.recipe.egg
+
+eggs =
+	isort
+	setuptools
+
+args = [
+    '--recursive',
+    '--check-only',
+    '--diff',
+    ]
+
+initialization =
+    import os
+    import subprocess
+
+    os.chdir("${buildout:directory}")
+
+    pkgdir = subprocess.Popen(
+        '${buildout:directory}/bin/package-directory relative',
+        shell=True, stdout=subprocess.PIPE,
+        ).stdout.read().strip()
+
+    if '--absolute' in sys.argv:
+        pkgdir = os.path.abspath(pkgdir)
+        sys.argv.remove('--absolute')
```

Comment:
Ah, that's actually there so that I can integrate well with my [emacs config](https://github.com/senny/cabbage/blob/jone/bundles/plone/buildout.el#L141-L142) 😎 
It makes pycodestyle output absolute paths instead of relative paths in its grep-style-output so that `python-pep8`-mode can parse it.

`isort` output is different anyway, so no need to keep that 😉